ietf/snippets/templates/snippets/disclaimer.html

```diff
@@ -4,29 +4,20 @@
     <div class="body-subtle">
         <div class="rule-blue body">
             <div class="container">
-                <h2>Note Well</h2>
                 <div>
-                    <p>Any submission to the IETF intended by the Contributor for publication as all or part of an IETF Internet-Draft or RFC and any statement made within the context of an IETF activity is considered an "IETF Contribution". Such statements include oral statements in IETF sessions, as well as written and electronic communications made at any time or place, which are addressed to:</p>
-
-                    <p>
-                        <ul>
-                            <li class="bullet-override">The IETF plenary session</li>
-                            <li class="bullet-override">The IESG, or any member thereof on behalf of the IESG</li>
-                            <li class="bullet-override">Any IETF mailing list, including the IETF list itself, any working group or design team list, or any other list functioning under IETF auspices</li>
-                            <li class="bullet-override">Any IETF working group or portion thereof</li>
-                            <li class="bullet-override">Any Birds of a Feather (BOF) session</li>
-                            <li class="bullet-override">The IAB or any member thereof on behalf of the IAB</li>
-                            <li class="bullet-override">The RFC Editor or the Internet-Drafts function</li>
-                            <li class="bullet-override">All IETF Contributions are subject to the rules of RFC 5378 and RFC 3979 (updated by RFC 4879).</li>
-                        </ul>
-                    </p>
-
-                    <p>Statements made outside of an IETF session, mailing list or other function, that are clearly not intended to be input to an IETF activity, group or function, are not IETF Contributions in the context of this notice.  Please consult RFC 5378 and RFC 3979 for details.</p>
-
-                    <p>A participant in any IETF activity is deemed to accept all IETF rules of process, as documented in Best Current Practices RFCs and IESG Statements.</p>
-
-                    <p>A participant in any IETF activity acknowledges that written, audio and video records of meetings may be made and may be available to the public.</p>
-
+                    {% if note_well %}
+                        <h2>{{ note_well.title }}</h2>
+                        {% if note_well.prepared_key_info %}
+                            {{ note_well.prepared_key_info|safe }}
+                        {% endif %}
+                        {% if note_well.prepared_in_depth %}
+                            {{ note_well.prepared_in_depth|safe }}
+                        {% endif %}
+                    {% else %}
+                        {# Fall back to some default text - copied from production note well at the time of writing #}
```

Comment:
Having a fallback is a good thing but I don't think showing a _possibly_ out of date version of note well text is a good idea.
Maybe add a link to the note well repository and contact details to let IETF support knows about the issue?
Something like https://notes.ietf.org/YXWF94BKRH-uqmQfRc-P1Q?view
@ghwood What do you think?

Reply:
Yes, I agree a pointer would be better.